pyfar/constants/constants.py

```diff
@@ -0,0 +1,302 @@
+        raise TypeError(
+            'relative_humidity must be a number or array of numbers')
+    if np.array(frequencies).ndim > 1:
+        raise ValueError('frequencies must be one dimensional.')
+    if not isinstance(
+            atmospheric_pressure, (int, float, np.ndarray, list, tuple)):
+        raise TypeError(
+            'atmospheric_pressure must be a number or array of numbers')
+
+    # check if broadcastable
+    try:
+        _ = np.broadcast_shapes(
+            np.atleast_1d(temperature).shape,
+            np.atleast_1d(relative_humidity).shape,
+            np.atleast_1d(atmospheric_pressure).shape)
+    except ValueError as e:
+        raise ValueError(
+            'temperature, relative_humidity, and atmospheric_pressure must '
+            'have the same shape or be broadcastable.') from e
+
+    # check limits
+    if np.any(np.array(temperature) < -73):
+        raise ValueError("Temperature must be greater than -73°C.")
+    if np.any(np.array(frequencies) < 50):
+        raise ValueError("frequencies must be greater than 50 Hz.")
+    if np.any(np.array(relative_humidity) < 0) or np.any(
+            np.array(relative_humidity) > 1):
+        raise ValueError("Relative humidity must be between 0 and 1.")
+    if np.any(np.array(atmospheric_pressure) > 200000):
+        raise ValueError("Atmospheric pressure must be less than 200 kPa.")
+    # convert arrays
+    temperature = np.array(
+        temperature, dtype=float)[..., np.newaxis]
+    relative_humidity = np.array(
+        relative_humidity, dtype=float)[..., np.newaxis]
+    atmospheric_pressure = np.array(
+        atmospheric_pressure, dtype=float)[..., np.newaxis]
+    frequencies = np.array(frequencies, dtype=float)
+
+    # calculate air attenuation
+    p_atmospheric_ref = pf.constants.reference_atmospheric_pressure
+    t_degree_ref = pf.constants.reference_air_temperature_celsius
+
+    h_r = relative_humidity*100
+    p_a = atmospheric_pressure
+    p_r = p_atmospheric_ref
+    f = frequencies
+    T = temperature - pf.constants.absolute_zero_celsius
+    T_0 = t_degree_ref - pf.constants.absolute_zero_celsius
+
+    # saturation vapour pressure
+    p_sat = _saturation_vapour_pressure_iso(temperature)
+
+    # molar concentration of water vapor as a percentage (Equation B.1)
+    h = h_r * (p_sat / p_r) * (p_a / p_r)
+
+    # Oxygen relaxation frequency (Eq. 3)
+    f_rO = (p_a/p_r)*(24+4.04e4*h*(0.02+h)/(0.391+h))
+
+    # Nitrogen relaxation frequency (Eq. 4)
+    f_rN = (p_a/p_r)*(T/T_0)**(-1/2)*(9+280*h*np.exp(
+        -4.17*((T/T_0)**(-1/3)-1)))
+
+    # air attenuation (Eq. 5)
+    air_attenuation = 8.686*f**2*((1.84e-11*p_r/p_a*(T/T_0)**(1/2)) + \
+        (T/T_0)**(-5/2)*(0.01275*np.exp(-2239.1/T)*(f_rO + (f**2/f_rO))**(-1)
+        +0.1068*np.exp(-3352/T) * (f_rN + (f**2/f_rN))**(-1)))
+
+    alpha = pf.FrequencyData(
+        air_attenuation, frequencies=frequencies)
+
+    m = alpha / (10*np.log10(np.exp(1)))
```

Comment:
The documentation states a multiplication instead of division. Is there a reference for this equation?